docs/src/workspaces/index.md

```diff
@@ -0,0 +1,7 @@
+# The Rust workspaces
+
+This section gives you a little overview of the role and function of every workspace in the 
+`fuelsrs` repository.
```

Comment:
```suggestion
This section gives you a little overview of the role and function of every workspace in the `fuels-rs` repository.
```

Reply:
Uh, sry I'm going to improve my markdown skills 😁